pkg/apply/merge.go

```diff
@@ -178,20 +178,23 @@ func mergeWebhookConfiguration(current, updated *unstructured.Unstructured) erro
 			continue
 		}
 
```

Comment:
is there a reason to keep this commit in this PR?

Reply:
+1, please add it to a separate one. The PR is huge as it is.